README.md

```
@@ -8,32 +8,47 @@ This project is licensed under the GNU General Public License v3.0. See the [LIC
+    REDIS_PORT={your_port}
+    REDIS_PASSWORD={your_password}
+    REDIS_DB=0                      # Redis logical database number (typically 0-15)
+    REDIS_PROTOCOL=3                # Redis protocol version (2 or 3)
+    # Others
+    PRODUCTION={true/false}
     ```
 
 2. Make sure you got [docker](https://www.docker.com/) installed.
 
-3. There's a `docker-compose.yml` file in the root directory, specifying the containers configurations. 
+3. There's a `docker-compose.yml` file in the root directory, specifying the containers' configurations. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)